### DISCUSSION TITLE
Removing folder/file while unable to access said path through github

### DISCUSSION THREAD

**Discussion by supermedman** [community/community#135434] [New to GitHub]
I am currently struggling to revert/delete a change that I made while creating issue templates. The issue at hand throws a 500 server error when attempting to view the .github folder path from any point in which it exists, preventing the removal of the files in question. How might one go about rectifying this? Im at quite a loss with the inability to create support tickets using the provided support page from the 500 server error display page. 

Edit: 
For context I am currently the sole repo owner and manager on a personal project. I am the only person with the ability to make and/or change anything within the repo.

[This is the path in question](https://github.com/supermedman/bbf/tree/main/.github/ISSUE_TEMPLATE)

**Comment by geniuszly** [comment]
Hi! To resolve the issue of being unable to access and delete a problematic folder or file in your GitHub repository due to a 500 server error, you can follow these steps: 
1. Open your terminal or command prompt. Clone your repository to your local machine using the following command and navigate to the cloned repository directory:
```bash
git clone https://github.com/supermedman/bbf.git
cd bbf
```
2. Once inside the repository directory, you can delete the `.github/ISSUE_TEMPLATE` directory locally using the following command:
```bash
rm -rf .github/ISSUE_TEMPLATE
```
Alternatively, you can manually navigate to the directory in your file explorer and delete it.
3. After deleting the folder or file, commit the changes locally:
```bash
git add .
git commit -m "Remove problematic ISSUE_TEMPLATE folder"
```
4. Push the changes back to your GitHub repository:
```bash
git push origin main
```
Ensure that "main" is the correct branch name; if your default branch is different, replace "main" with the appropriate branch name.
5. Go back to your GitHub repository in your web browser. Check that the `.github/ISSUE_TEMPLATE` folder has been removed and that you no longer encounter the 500 server error.
6. If you are using GitHub Pages or any automation tools linked with your repository, ensure that the removal of the folder has not affected the functionality of those services.

**Reply by supermedman** [reply]
This worked as needed, all of my git repo was already on my pc to start as I locally host the bot that the repo contains. Making the clone in my downloads folder (To avoid conflicts with OneDrive auto uploads) I was able to manually remove the .yml file that caused issue. Thank you for the assistance with this!!

**Reply by geniuszly** [reply]
I was glad to help! If you have any problems, you can find my contacts in my profile and contact me if anything happens.

**Reply by supermedman** [reply]
Will do, thanks again! <3
